enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -2263,6 +2276,12 @@ def test_can_redeem_policy_beyond_enrollment_deadline(self, mock_lms_client, moc
             "content_price": 1234,
             "enroll_by_date": '2001-01-01T00:00:00Z',
         }
+        self.mock_catalog_get_and_cache_content_metadata.return_value = {
+            'normalized_metadata': {
+                'content_price': 12.34,  # normalized_metadata always serializes USD.
```

Comment:
nit: should this be an integer vs. float?

[Example](https://enterprise-catalog.edx.org/api/v1/content-metadata/TheEconomist+CRIT/?format=json)

```
{
  "normalized_metadata": {
    "start_date": "2025-03-10T00:00:00Z",
    "end_date": "2025-03-21T00:00:00Z",
    "enroll_by_date": "2025-03-16T00:00:00Z",
    "enroll_start_date": null,
    "content_price": 828
  },
}
```